tests/test_unit/brainglobe_workflows/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+
+@pytest.fixture()
+def custom_logger_name() -> str:
+    """Return name of custom logger created in workflow utils
+
+    Returns
+    -------
+    str
+        Name of custom logger
+    """
+    from brainglobe_workflows.utils import __name__ as LOGGER_NAME
+
+    return LOGGER_NAME
```

Comment:
I liked the second superconcise suggestion but Python won't take it (I get `brainglobe_workflows not defined`)

> Or move this fixture inside the test file where it's used (or even inside the code, and don't use a fixture at all?)

I thought having all tests fetch the logger name from the same source (instead of assuming the name is the module name) could be convenient if we later choose to change its name. But we do get another `contest.py` file for one fixture only. Not feeling very strongly either way, so happy to do whatever you prefer!

Reply:
I will leave this as is for now but [added a comment](https://github.com/brainglobe/brainglobe-workflows/issues/25) in a logger issue in case we want to reconsider in the future